test/node.js

```diff
@@ -0,0 +1,126 @@
+import { expect } from "chai";
+import fs from "fs";
+import os from "os";
+import path from "path";
+import process from "process";
+import { promisify } from "util";
+
+describe("Errors", function () {
+  it("Should return error", function () {
+    function throwError() {
+      throw new Error("oh no!");
+    }
+    expect(() => throwError()).to.throw("oh no!");
+  });
+});
+
+describe("OS", function () {
+  it("Should use os.arch()", function () {
+    expect(os.arch()).to.equal("x64");
+  });
+  it("Should use os.homedir()", function () {
+    console.log(os.homedir());
+  });
+});
+
+describe("Path", function () {
+  it("Should use path.basename()", function () {
+    expect(path.basename("/foo/bar/baz/asdf/quux.html")).to.equal("quux.html");
+  });
+  it("Should use path.dirname()", function () {
+    expect(path.dirname("/foo/bar/baz/asdf/quux.html")).to.equal(
+      "/foo/bar/baz/asdf"
+    );
+  });
+  it("Should use path.format()", function () {
+    const pathObj = {
+      root: "/ignored",
+      dir: "/home/user/dir",
+      base: "file.txt",
+    };
+    expect(path.format(pathObj)).to.equal("/home/user/dir/file.txt");
+  });
+  it("Should use path.isAbsolute()", function () {
+    expect(path.isAbsolute("/foo/bar")).to.equal(true);
+    expect(path.isAbsolute("/baz/..")).to.equal(true);
+    expect(path.isAbsolute("qux/")).to.equal(false);
+    expect(path.isAbsolute(".")).to.equal(false);
+  });
+});
```

Comment:
I suggest that you also add some tests to explore `path.join()`

Reply:
Added